ee/tabby-webserver/src/service/auth.rs

```diff
@@ -173,19 +177,57 @@ impl AuthenticationService for DbConn {
             return Err(TokenAuthError::InvalidPassword);
         }
 
+        let utc_ts = chrono::Utc::now().timestamp();
+        let refresh_token = match self.get_user_unexpired_token(user.id, utc_ts).await? {
```

Comment:
does it makes more sense to always return a new token? (So different login can be distinguished and revoked individually)

Reply:
hmm...so this is expected behavior?

Reply:
Otherwise a client can just refresh access token without update its local refresh token?

Reply:
note this somewhat coupled with the behavior above (always update the token with refreshing).

Here we should always create a new refresh token for each login.